.github/workflows/wpt_chromedriver.yml

```diff
@@ -0,0 +1,83 @@
+# This workflow will check out wpt and run the WebDriver BiDi tests against our
+# implementation.
+
+name: Web platform tests with ChromeDriver
+
+on:
+  pull_request:
+  push:
+    branches: 'main'
+
+jobs:
+  wpt:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Set up node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: npm
+      - name: Install and build npm dependencies
+        run: npm ci
+      - name: Setup dirs
+        run: |
+          mkdir -p logs wpt-metadata/chromeDriver
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+          cache: 'pip'
+      - name: Set up virtualenv
+        run: pip install virtualenv
+      - name: Set up hosts
+        run: ./wpt make-hosts-file | sudo tee -a /etc/hosts
+        working-directory: wpt
+      # TODO: Install a pinned version of Chromium. This may become possible
+      # after https://github.com/web-platform-tests/wpt/issues/28970.
+      - name: Install Chromium
+        # This installs dev chrome to `/usr/bin/google-chrome-unstable`.
+        run: |
+          wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+          sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-unstable
+      - name: Run tests
+        run: >
+          ./wpt/wpt run 
+          --binary-arg="--headless=new" 
+          --webdriver-arg="--bidi-mapper-path=lib/iife/mapperTab.js"  
+          --webdriver-arg="--log-path=logs/chromeDriver.log" 
+          --webdriver-arg="--verbose" 
+          --install-webdriver --channel=dev --yes
+          --manifest MANIFEST.json 
+          --metadata wpt-metadata/chromeDriver  
+          --log-wptreport wpt/wptreport.json 
+          chrome 
+          webdriver/tests/bidi/
+      - name: Update expectations
```

Comment:
The main difference is this step.
It's possible to keep this within a single workflow file and reuse smaller/common steps, with dependent jobs.
This can be submitted as-is for know, but we could follow-up by merging them.

Reply:
Ideally we want to cache the dependencies: https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows